dev-db/sqlitestudio/sqlitestudio-3.4.3.ebuild

```diff
@@ -132,6 +164,12 @@ src_compile() {
 }
 
 src_install() {
+	if use test; then
+		# remove test artifacts that must not be installed
+		rm -r "${ED}"/lib64 || die
+		rm -r "${ED}"/usr/share/qt5/tests || die
+	fi
```

Comment:
This looks good now, but it fails.  Cannot remove files before installing them, this part should go below emake install.  I can do that while merging.

As for a style guide, I'm not sure we have one other than the [devmanual](https://devmanual.gentoo.org/)